d3-cloud/resources/cljsjs/d3-cloud/common/d3-cloud.ext.js

```diff
@@ -0,0 +1,2 @@
+var d3 = {};
+d3.layout.cloud = function () {};
```

Comment:
This might fail because `d3.layout` is null. Probably it should be defined as empty object.